README.md

```diff
@@ -176,14 +176,73 @@ let nftType = nft.getType()
+| `Editions`  | Wrapper for multiple edition views.        | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L176-L187)|
+| `Serial`    |                                            | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L200-L211)|
+| `Royalty`   | A Royalty Cut for a given NFT.             | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L224-L264) |
+| `Royalties` | Wrapper for multiple Royalty views.        | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L266-L290) |
+| `Media`     |                                            | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L311-L328)|
+| `Medias`    |                                            | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L330-L340)|
+| `License`   |                                            | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L353-L362)|
+| `ExternalURL`|                                           | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L375-L385)|
+| `NFTCollectionData` | Provides storage and retrieval information of an NFT | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L398-L455) |
+| `NFTCollectionDisplay` | Returns the basic representation of an NFT's Collection.  | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L468-L507) |
+| `Rarity`   |                                            | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L523-L550)|
+| `Trait`    |                                            | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L563-L590)|
+| `Traits`   |                                            | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L592-L605)|
+
+## Allways prefer wrappers over single views
+
+When exposing a view that could have multiple ocurrences on a single NFT, such as `Edition`, `Royalty`, `Media` or `Trait` the wrapper view should allways be exposed, even if there is a single occurrence.
+
+When resolving the view, the wrapper view should be the returned value, instead of returning the single view or just an array of several ocurrences of the view
```

Comment:
```suggestion
When resolving the view, the wrapper view should be the returned value, instead of returning the single view or just an array of several occurrences of the view.
```